packages/blade/src/components/Skeleton/Skeleton.stories.tsx

```diff
@@ -0,0 +1,424 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+import { Skeleton, Skeleton as SkeletonComponent } from './Skeleton';
+import type { SkeletonProps } from './Skeleton';
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardHeaderBadge,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+} from '~components/Card';
+import { Box } from '~components/Box';
+import { Code, Heading, Text, Title } from '~components/Typography';
+import { Amount } from '~components/Amount';
+import { Button } from '~components/Button';
+import { Alert } from '~components/Alert';
+import { getBoxArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import { Divider } from '~components/Divider';
+import { announce } from '~components/LiveAnnouncer';
+import { List, ListItem, ListItemCode } from '~components/List';
+import { isReactNative } from '~utils';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=16508%3A258522',
+        bankingTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=16508%3A258522',
+      }}
+      componentName="Skeleton"
+      componentDescription="Skeleton Loader is a static / animated placeholder for the information that is still loading. It mimic the structure and look of the entire view."
+    >
+      <Title>Usage</Title>
+      <Sandbox>
+        {`
+        import { Skeleton } from '@razorpay/blade/components';
+        
+        function App(): JSX.Element {
+          return (
+            <Skeleton width="100%" height="50px" margin="spacing.4" />
+          )
+        }
+
+        export default App;
+        `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+const argTypes = getBoxArgTypes();
+const filteredArgs = Object.fromEntries(
+  Object.entries(argTypes)
+    .filter(
+      ([key]) =>
+        !key.includes('border') &&
+        !key.includes('background') &&
+        !key.includes('overflow') &&
+        !['as', 'textAlign', 'tabIndex', '__brand__'].includes(key),
+    )
+    .map(([key, _value]) => {
+      return [key, _value];
+    }),
+);
+
+export default {
+  title: 'Components/Skeleton',
+  component: SkeletonComponent,
+  argTypes: filteredArgs,
```

Comment:
I can see all of the values regardless of this. Can you check this once? We're supporting very limited set of styling props here

Reply:
In general, the sorting is a bit messed up. Component-specific props have disappeared from the list. Just check the storybook once for the list of properties and see if we can clean it up a bit

Reply:
Updated check once.